torch/linalg/__init__.py

```diff
@@ -234,10 +234,30 @@
     Consider using :func:`torch.linalg.solve` if possible for multiplying a matrix on the left by
     the inverse, as::
 
-        torch.linalg.solve(A, B) == A.inv() @ B
+        torch.linalg.solve(A, B) == torch.linalg.inv(A) @ B  # When B is a matrix
```

Comment:
I think simply adding the comment  `# When B is a matrix` + the modification you did of `inv` -> `torch.linalg.inv` already does the job. Note that this is a note in the `inv` function, in this case, `B` will always be a matrix.

As a very minor note, what do you think about going from `torch.linalg.solve` to `linalg.solve` here (same for `inv`)? I think it's more concise. You could then change the reference above to ``:func:`~linalg.solve` ``

Reply:
Thank you, yes I agree with your comments, I wasn't sure how much of an explanation was needed, I'll change linalg.tensorsolve accordingly and update the commit.

Also it was mentioned in the issue that the error #71385 that the matmul error thrown is potentially not as descriptive:
* Fix error message (as torch.linalg.inv(A).shape is [2, 3, 3])

since matmul throws 
```
# RuntimeError: mat1 and mat2 shapes cannot be multiplied (6x3 and 2x3)
```
but torch.linalg.inv(A).shape is [2, 3, 3]

Just wondering if you feel like that should also be included to resolve

Reply:
I think that one's going to be tricky, as I don't think that there's much that can be done there given how matmul works. I'd say it's fine to mark both issues as resolved once we update the docs.

Reply:
Done! I decided to leave the top reference as the full path because its explicit and consistent with the `tensorsolve` doc Therefore for tensorsolve I added the comment in the example and that's it. For inv I changed the example and shortened the references to `lingal.solve` and `lingal.inv` respectively.